CONTRIBUTING.md

```diff
@@ -98,7 +93,42 @@ Right after opening project, Visual Studio will read cmake files and generate ca
 
 ### Linux / Mac
 
-VCPKG can also be used in Linux to avoid installing libraries globally. Follow instructions [here](#vcpkg) to use VCPKG in Linux.
+#### VCPKG
+
+VCPKG can be used to download packages sources, headers and build libraries for whatever TRIPLET is set up (platform/arq).
+VCPKG maintains any installed package inside its own folder, allowing to have multiple vcpkg folder with different dependencies installed on each. This is also great because you don't have to install dependencies globally on your system.
+
+Follow next steps to install VCPKG and have it linked to cmake
+
+```bash
+# Clone vcpgk:
+git clone https://github.com/Microsoft/vcpkg.git
+# (consider this path as PATH_TO_VCPKG)
+cd vcpkg
+# build vcpkg (remove .bat on Linux/Mac)
+./bootstrap-vcpkg.sh
```

Comment:
comment and instructions mismatch. there is no `.bat` here. Change the command, or update comment to say use `.bat` on Windows.